src/main/java/core/basesyntax/service/FruitOperationImpl.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service;
+
+import core.basesyntax.entity.Fruit;
+import core.basesyntax.db.FruitStorage;
+
+public class FruitOperationImpl implements FruitOperation {
+    @Override
+    public void transaction(Fruit fruit) {
```

Comment:
create method 'add' in FruitStorage and use it for save fruits in all places. This class can be removed